README.md

```diff
@@ -119,7 +119,7 @@ See [Contributing](.github/CONTRIBUTING.md).
 
 [Analytics](https://en.wikipedia.org/wiki/Analytics) is the systematic computational analysis of data or statistics. It is used for the discovery, interpretation, and communication of meaningful patterns in data.
 
-**Please visit [Awesome Analytics](https://github.com/newTendermint/awesome-analytics)**
+**Please visit [Awesome Analytics](https://github.com/0xnr/awesome-analytics)**
```

Comment:
This commit (89c4d8e177a793db460c110a540986490f543021) accidentally reverts a bunch of changes. Can you please drop this commit?

Reply:
626400b86ca03c43cbacd16ada92ed8a25094cdb and following commits appear to fix the problem.

Reply:
Yeah haha, No idea why the merge reverted alot 😆